src/main/java/frc/robot/commands/balancing/DefaultBalancerCommand.java

```diff
@@ -49,6 +53,12 @@ public void execute() {
                 reachedStation = true;
                 returnDrivePower = -0.17;
             }
+            else if(runawayTimer.hasElapsed(2)){
+                returnDrivePower = 0.0;
+                System.out.println("BALANCER RUNAWAY DETECTED");
+                end(false);
```

Comment:
Calling `end(false)` won't actually end the command. You should set whatever `isFinished()` returns to `true`, and then the command scheduler will automatically end the command and call the end method.